integration

```diff
@@ -249,6 +249,7 @@
   "cagnulein/switchbot_press",
   "caiosweet/Home-Assistant-custom-components-DPC-Alert",
   "caiosweet/Home-Assistant-custom-components-INGV",
+  "carohauta/oma-helen-ha-integration",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>

Reply:
That is a good point. Added the country code and created a new release.

Link to current release: https://github.com/carohauta/oma-helen-ha-integration/releases/tag/2.0.1
Link to successful HACS action (without the ignore key): https://github.com/carohauta/oma-helen-ha-integration/actions/runs/17788251898
Link to successful hassfest action (if integration): https://github.com/carohauta/oma-helen-ha-integration/actions/runs/17788260544